loopy/codegen/__init__.py

```diff
@@ -204,6 +206,10 @@ class CodeGenerationState:
         generated.
 
     .. attribute:: schedule_index_end
+
+    .. attribute:: codegen_cachemanager
```

Comment:
```suggestion
    .. attribute:: codegen_cache_manager
```
For consistency with the capitalization in the class name.